content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -230,15 +240,20 @@ clusters, zones, or regions.
 This model affords the following benefits:
 
 - Improved availability: If a control plane becomes unavailable, the scope of
-  the outage is limited to only that control plane.
+  the outage is limited to only that control plane, and the workloads in clusters managed by that control plane.
```

Comment:
```suggestion
  the outage is limited to only workloads in clusters managed by that control plane.
```

Reply:
is there workload outage when istio config/resources aren't changed?  Seems odd we would highlight workloads outage here.